content/6_Advanced/Wavelet.mdx

```diff
@@ -1,19 +1,17 @@
 ---
 id: wavelet
 title: 'Wavelet Tree'
-author: Benjamin Qi
+author: Benjamin Qi, Crafticat
 prerequisites:
   - RURQ
-description: "?"
+description: Segment tree over values"
 frequency: 0
 ---
 
-## Wavelet Tree
 
-<FocusProblem problem="waveletSam" />
-
-Like a segment tree on values rather than indices.
+# Wavelet Tree
```

Comment:
i presume you've followed the [module layout guidelines](https://docs.google.com/document/d/1Dzj5uYNYNXe_E-_9-lSeX73654cGJk88PZ0kZCCmks0/edit)?

Reply:
not addressed?

Reply:
Yes?

Reply:
no yeah this mostly works